### PR TITLE
[rpc] Descriptions: Consistent arg labels for types 'object', 'array', 'boolean', and 'string'

### PR DESCRIPTION
**Pull request by ch4ot1c**
Followup for https://github.com/bitcoin/bitcoin/pull/14459. See `scripted-diff:` commit messages for scripts used.

### CODE REVIEW

File: src/rpc/mining.cpp [hunk 1]
```diff
@@ -878,23 +878,23 @@ static UniValue estimaterawfee(const JSONRPCRequest& request)
...
             "          \"endrange\" : x.x,       (numeric) end of feerate range\n"
             "          \"withintarget\" : x.x,   (numeric) number of txs over history horizon in the feerate range that were confirmed within target\n"
             "          \"totalconfirmed\" : x.x, (numeric) number of txs over history horizon in the feerate range that were confirmed at any point\n"
             "          \"inmempool\" : x.x,      (numeric) current number of txs in mempool in the feerate range unconfirmed for at least target blocks\n"
             "          \"leftmempool\" : x.x,    (numeric) number of txs over history horizon in the feerate range that left mempool unconfirmed after target\n"
             "      },\n"
-            "      \"fail\" : { ... },       (json object, optional) information about the highest range of feerates to fail to meet the threshold\n"
+            "      \"fail\" : { ... },       (object, optional) information about the highest range of feerates to fail to meet the threshold\n"
             "      \"errors\":  [ str... ]   (json array of strings, optional) Errors encountered during processing\n"
```
Comment (MarcoFalke) [inline src/rpc/mining.cpp]: Could remove the json here as well?

**Review by maflcko — COMMENTED** [review]
Concept ACK on making this consistent. Later improvements to make the documentation machine generated would be easier to review.

I don't have an opinion on the actual type name, but it should be consistent.

**Comment by MarcoFalke** [comment]
Needs rebase, but should be simple for a scripted-diff. Also note that RPCHelpMan uses "json array" and "json object", as opposed to just "object"/"array".

**Comment by ch4ot1c** [comment]
Closing in favor of #14459.

**Pull request closed without merging**